app/src/androidTest/java/com/github/dedis/student20_pop/ui/HomeFragmentTest.java

```diff
@@ -42,20 +50,32 @@ public void homeFragmentIsDisplayed(){
     @Test
     public void listOfLaosIsDisplayed(){
         onView(withId(R.id.lao_list)).check(matches(isDisplayed()));
-        onData(allOf(is(instanceOf(Lao.class)))).atPosition(0).check(matches(isDisplayed()));
-        onData(allOf(is(instanceOf(Lao.class)))).atPosition(1).check(matches(isDisplayed()));
-        onData(allOf(is(instanceOf(Lao.class)))).atPosition(2).check(matches(isDisplayed()));
+    }
+
+    public static Matcher<Object> titleMatches(String title){
```

Comment:
Maybe it could be nice to have a short description of what this Matcher is used for